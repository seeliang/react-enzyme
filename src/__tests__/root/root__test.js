
import React from 'react';
import renderer from 'react-test-renderer';
import {it, expect} from 'jest';

import Enzyme from 'enzyme';
import {mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Root from '../../root';

Enzyme.configure({ adapter: new Adapter() });

it('matches snapshot', () => {
  const tree = renderer.create(
    <Root/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('has input',() => {
  const tree = mount(
    <Root/>
  );
  expect(tree.find('input')).toHaveLength(1);
});
