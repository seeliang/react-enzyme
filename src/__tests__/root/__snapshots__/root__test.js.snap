// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`some basics shall matches snapshot 1`] = `
<div>
  <h1>
     Hello world
  </h1>
  <input
    type="checkbox"
  />
  <button
    onClick={[Function]}
  >
     try this
  </button>
</div>
`;
